machine.md

```diff
@@ -4,6 +4,41 @@ Finetuning different aspects of the hardware which are not device related
 (BIOS, mainboard, ...) is sometimes necessary to allow Guest Operating Systems
 to properly boot and reboot.
 
+## Machine Type
+
+QEMU is able to work with two different classes of chipsets for x86\_64, so
```

Comment:
Is this shorthand for "x86 and x86_64"? I haven't seen that before.

Reply:
No, it should be markdown formatting thingy. Will check if it's needed.

Reply:
http://johnmacfarlane.net/babelmark2/?text=my%5C_name%5C_%0A%0Amy_name_ this example from SO indicates that it is the most supported underscore formatting available; in the final processed document only underscore itself is displayed.